### DISCUSSION TITLE
"rememberable" in the context of this gem

### DISCUSSION THREAD

**Discussion by tinynumbers** [graphql-devise/graphql_devise#254] [Q&A]
The examples show the use of the `rememberable` Devise module in the example User models, but it is unclear to me how (or if?) "remember me" behavior is handled by this gem. There's not a `remember_me` argument on the user login mutation.

How would we support a "remember me" checkbox on a user login form using `graphql_devise`?

**Comment by mcelicalderon** [comment]
Hey, @tinynumbers! Honestly I have focused mostly on the token based authentication for this gem since I think that might be the most common use case for an API (authenticate each request with a token). But I'm curious of your use case, are you using this gem with session based cookies? `Rememberable` works with cookies, and I know we do handle sessions in cookies in this gem too, so I will take a closer look at this. Anyway, if you could provide a bit more of context on how you would like to use rememberable, that would be awesome. Just note that rememberable by default remembers a user for 2 weeks, and that's something you could accomplish with tokens simply by making the token expiration to be 2 weeks. One thing I do think the rememberable module supports that tokens won't allow right now, is extending the remeberable period whenever the user becomes active again

**Reply by tinynumbers** [reply]
I'm experimenting with using `graphql_devise` in a React frontend that integrates with Rails via [`vite_rails`](https://vite-ruby.netlify.app/guide/rails.html). And the plan is to deploy the UI and Rails backend to the same domain. That opens me up to cookie-based auth, which IMO is easier and more secure than dealing with tokens.

**Reply by mcelicalderon** [reply]
Thank you, that makes sense. From a quick look I think using the rememberable module should be possible when using cookie based authentication. I'll take a closer look soon to see if this is something we can add support for in this gem
